### ISSUE TITLE
Implement open contracting id in contract summary page

### ISSUE THREAD

**Issue by anderspeders** [NRGI/resourcecontracts.org#340]
This is pending on implementation of #173

**Comment by anjesh** [comment]
Yes. But we have placed the implementation in the codebase but the structure is not followed yet, so not deployed. We will have this on Monday. Also we need to build the script to generate the OCID for those existing contracts, and also have OCID as a part of the contract url in the subsites.

**Comment by anderspeders** [comment]
excellent, thanks

**Comment by anderspeders** [comment]
Ok - given that OCDS ID has now been added, I will close this ticket. 

I will add a new ticket for adding the OCDS ID to the URL - which will be scheduled for post-OLC launch.
